### ISSUE TITLE
Test deprecation warning

### ISSUE THREAD

**Issue by jlaehne** [LumiSpy/lumispy#54]
#44 introduced the following deprecation warning, which we should fix:
```
lumispy/tests/test_luminescence_spectrum.py::TestLumiSpectrum::test_errors_raise
  /home/jonas/medien/pdi/git/lumispy/lumispy/signals/luminescence_spectrum.py:312: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
    background_xy = np.array(background)
```

**Comment by jordiferrero** [comment]
@jordiferrero Will do asap.
Sorry

**Comment by jlaehne** [comment]
Just wanted to document it, before we forget it.

**Comment by jlaehne** [comment]
Just found another warning in the test suite, which I think results from https://github.com/LumiSpy/lumispy/blob/b28238198f52cb52c6557232cb49ab2175ab37e8/lumispy/tests/test_axis-conversion.py#L305

I guess that `solve_grating_equation(axis2, 1, 1, 1, 1, 1, 1)` triggers the following warning when running `_n_air()`

```
lumispy/tests/utils/test_axis-conversion.py::test_solve_grating_equation
  /home/jonas/medien/pdi/git/lumispy/lumispy/utils/axes.py:51: UserWarning: The wavelength range exceeds the interval of 185 to 1700 nm for which the exact refractive index of air is used. Beyond this range, the refractive index is kept constant.
    warn("The wavelength range exceeds the interval of 185 to 1700 nm for "
```

**Comment by jordiferrero** [comment]
> Just found another warning in the test suite, which I think results from
> 
> https://github.com/LumiSpy/lumispy/blob/b28238198f52cb52c6557232cb49ab2175ab37e8/lumispy/tests/test_axis-conversion.py#L305
> 
> I guess that `solve_grating_equation(axis2, 1, 1, 1, 1, 1, 1)` triggers the following warning when running `_n_air()`
> 
> ```
> lumispy/tests/utils/test_axis-conversion.py::test_solve_grating_equation
>   /home/jonas/medien/pdi/git/lumispy/lumispy/utils/axes.py:51: UserWarning: The wavelength range exceeds the interval of 185 to 1700 nm for which the exact refractive index of air is used. Beyond this range, the refractive index is kept constant.
>     warn("The wavelength range exceeds the interval of 185 to 1700 nm for "
> ```

It does. But I wanted to test for that. I guess I will have to add the warning test on it so it accounts it as a warning within the test and not as an unplanned warning. Noted.

**Comment by jlaehne** [comment]
@jordiferrero could you have a look at these warnings? Then we could maybe do a patch release soon.

**Comment by jlaehne** [comment]
> > Just found another warning in the test suite, which I think results from
> > https://github.com/LumiSpy/lumispy/blob/b28238198f52cb52c6557232cb49ab2175ab37e8/lumispy/tests/test_axis-conversion.py#L305
> > 
> > I guess that `solve_grating_equation(axis2, 1, 1, 1, 1, 1, 1)` triggers the following warning when running `_n_air()`
> > ```
> > lumispy/tests/utils/test_axis-conversion.py::test_solve_grating_equation
> >   /home/jonas/medien/pdi/git/lumispy/lumispy/utils/axes.py:51: UserWarning: The wavelength range exceeds the interval of 185 to 1700 nm for which the exact refractive index of air is used. Beyond this range, the refractive index is kept constant.
> >     warn("The wavelength range exceeds the interval of 185 to 1700 nm for "
> > ```
> 
> It does. But I wanted to test for that. I guess I will have to add the warning test on it so it accounts it as a warning within the test and not as an unplanned warning. Noted.

This one is fixed in #104 , the deprectation warning remains to be done
